architecture/Solution Architecture.md

```diff
@@ -0,0 +1,314 @@
+# COVID-19 Notification App - Solution Architecture
+
+## Baseline for the "Proof of Concept"
+
+**Version:** 0.6 (28/5/2020)
+
+**Status: ** Draft
```

Comment:
Should be `**Status:**` (no space), see [this markdown online](https://github.com/ijansch/nl-covid19-notification-app-coordination/blob/initial_version/architecture/Solution%20Architecture.md) to see the current rendering.

Reply:
Thanks, fixed